docs/guides/high-availability/deploy-aws-accelerator-loadbalancer.adoc

```diff
@@ -0,0 +1,294 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Deploy an AWS Global Accelerator loadbalancer"
+summary="Building block for a loadbalancer"
+tileVisible="false" >
+
+This topic describes the procedure required to deploy an AWS global accelerator to route traffic between multi-site Keycloak deployments.
+
+include::partials/blueprint-disclaimer.adoc[]
+
+== Audience
+
+This guide describes how to deploy an AWS Global Accelerator instance to handle Keycloak client connection failover for multiple
```

Comment:
This also applies to all other places, please review. The more or less only place where it is allowed is "Keycloak CR".

```suggestion
This guide describes how to deploy an AWS Global Accelerator instance to handle {project_name} client connection failover for multiple
```